### PR TITLE
User Interface changes

### PR DESCRIPTION
**Pull request by putera**
Dropdown width :

Current Situation :

Fixed

Notification in HTML - Allow HTML in notification :

Able to Show/Hide Selected Items on the Select component :

```blade
<x-select
        label="Select Users"
        placeholder="..."
        wire:model.lazy="model"
        :async-data="route('users')"
        option-label="name"
        option-value="id"
        multiselect
        :show-items-count="false"
    />
```

### CODE REVIEW

**Comment by ph7jack** [comment]
@putera Great feature, mark me when it's ready to merge

**Comment by putera** [comment]
> @putera Great feature, mark me when it's ready to merge

Will do bro. 👍🏿👍🏿👍🏿

**Comment by ph7jack** [comment]
@putera if you need some help with this feature, you can ask me. This PR is very useful
Thanks bro

**Comment by putera** [comment]
> @putera if you need some help with this feature, you can ask me. This PR is very useful
> Thanks bro

Ok brother, i will proceed with this first.

File: resources/views/components/button.blade.php [hunk 1]
```diff
@@ -4,6 +4,7 @@
     $defaultAttributes = [
         'wire:loading.attr'  => 'disabled',
         'wire:loading.class' => '!cursor-wait',
+        'wire:target' => (strlen($spinner) > 1) ? $spinner : null,
```
Comment (ph7jack) [inline resources/views/components/button.blade.php]:
```suggestion
        'wire:loading.attr'  => 'disabled',
        'wire:loading.class' => '!cursor-wait',
        'wire:target'        => (strlen($spinner) > 1) ? $spinner : null,
```

File: resources/views/components/dropdown.blade.php [hunk 2]
```diff
@@ -23,7 +23,7 @@ class="w-4 h-4 text-secondary-500 hover:text-secondary-700
         x-transition:leave="transition ease-in duration-75"
         x-transition:leave-start="opacity-100 scale-100"
         x-transition:leave-end="opacity-0 scale-95"
-        class="{{ $getAlign() }} z-30 absolute mt-2 w-56"
+        class="{{ $getAlign() }} z-30 absolute mt-2 whitespace-nowrap"
```
Comment (ph7jack) [inline resources/views/components/dropdown.blade.php]:
```suggestion
        class="{{ $getAlign() }} z-30 absolute mt-2 {{ $width }} whitespace-nowrap"
```

File: src/View/Components/Select.php [hunk 3]
```diff
@@ -11,6 +11,7 @@ public function __construct(
         public bool $clearable = true,
         public bool $searchable = true,
         public bool $multiselect = false,
+        public bool $showItemsCount = true,
```
Comment (ph7jack) [inline src/View/Components/Select.php]: Could you rename it to **withoutItemsCount**, so we can pass it in an easy way
```suggestion
        public bool $withoutItemsCount = false,
```
`<x-select without-items-count ... />`

**Review by ph7jack — CHANGES_REQUESTED** [review]
Amazing work ❤️ 

Could you fix these comments?

**Review by ph7jack — APPROVED** [review]
Thanks a lot for contributing

**Pull request merged**